### ISSUE TITLE
Test: accessibility detection improvements

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#167159]
Refs: https://github.com/microsoft/vscode/issues/162331

- [x] Windows @hediet
- [x] macOS @kieferrm
- [x] Linux @roblourens

Complexity: 3
Authors: @deepak1556 @isidorn 

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23167159%0A%0A&assignees=isidorn,deepak1556)

---

We have improved how we auto detect if a screen reader is on when using VS Code. Now this detection should be more precise. You should verify that the detection still works fine.
If you got Windows install [NVDA](https://www.nvaccess.org/download/) and [JAWS](https://www.freedomscientific.com/products/software/jaws/) and test both. If you got macOS - just use VoiceOver (Cmd+F5). If you got Linux - use [Orca](https://help.gnome.org/users/orca/stable/index.html.en).

* When starting a Screen Reader with VS Code running, all VS Code windows detect this and the status bar shows "Screen Reader Detected". The editor now begins to output screen reader content to the `<textarea>`. This is proved by the screen reader reading the text in the editor as the editor cursor moves. 
* Please also check other scenarios, such as first starting the screen reader and then launching VS Code, or reloading a window, or opening a new window, etc. In all cases, the fact that a screen reader is attached should be detected. This is important, because otherwise the editor does not output screen reader content to the `<textarea>` (which gives us a nice performance boost)
* Turn off screen reader. Start VS Code again. Make sure the screen reader is not detected (there is no status bar message).

**Comment by kieferrm** [comment]
This doesn't work for me at all. See https://github.com/microsoft/vscode/issues/167627.

**Comment by roblourens** [comment]
It's also not working with Orca. But Daniel suggested that Orca doesn't work well with Electron anyway?

**Comment by isidorn** [comment]
@kieferrm thanks for testing, let's continue the discussion in https://github.com/microsoft/vscode/issues/167627

@roblourens thanks for trying on Linux. I thought that later Chromium versions detect Orca better and would expect auto detection to work. @joanmarie might you know if latest Chrome versions still require the accessibility flag to be explicitly turned on (i.e they do not detect Orca)? Rob can you file an issue for this so we have this tracked?

fyi @meganrogge
